src/Illuminate/Events/functions.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Illuminate\Events;
+
+use Closure;
+
+/**
+ * Create a new queued Closure event listener.
+ *
+ * @param  \Closure  $closure
+ * @return \Illuminate\Events\QueuedClosure
+ */
+function queueable(Closure $closure)
```

Comment:
This can be side effect?

Can be like?;

if (! method_exists('queuable')) {
    // method here
}

Reply:
It's namespaced.

Reply:
You still need the if method exists, actually. The function can still conflict with a global install of itself.